pyanaconda/modules/storage/devicetree/fsset.py

```diff
@@ -20,6 +20,7 @@
 import time
 
 from blivet import blockdev
+from blivet.fstab import FSTabManager
```

Comment:
Please also bump required version of blivet in spec to the version adding fstab support.